BUILD.md

```diff
@@ -39,3 +40,12 @@ Developers who can work a command line may be interested in using [Visual Studio
 - Run `cmake . -G "Ninja"` to generate build files.
 
 - Run `cmake --build .` to build the project.
+  
+## Linux
+### Steps
+1. Clone the GitHub repo
+2. Use `cd` to navigate to the cloned repo's directory
+3. Then, run the following commands in order:
+* `docker build --rm -t test-fedora .`
+* `docker run --rm -it -e CC=cl -e CXX=cl --mount type=bind,source="$(pwd)",destination=/build test-fedora cmake . -DCMAKE_BUILD_TYPE=Release -DCMAKE_SYSTEM_NAME=Windows -G "Ninja"`
+* `docker run --rm -it -e CC=cl -e CXX=cl --mount type=bind,source="$(pwd)",destination=/build test-fedora cmake --build .`
```

Comment:
I would rename the image from `test-fedora` to something like `northstar-build-fedora` or something along the lines ^^

Reply:
```suggestion
* `docker build --rm -t northstar-build-fedora .`
* `docker run --rm -it -e CC=cl -e CXX=cl --mount type=bind,source="$(pwd)",destination=/build northstar-build-fedora cmake . -DCMAKE_BUILD_TYPE=Release -DCMAKE_SYSTEM_NAME=Windows -G "Ninja"`
* `docker run --rm -it -e CC=cl -e CXX=cl --mount type=bind,source="$(pwd)",destination=/build northstar-build-fedora cmake --build .`
```
make a suggestion then